app/src/main/java/com/sdp13epfl2021/projmag/activities/ProjectsListActivity.kt

```diff
@@ -22,12 +22,20 @@ import com.sdp13epfl2021.projmag.database.Utils
 import com.sdp13epfl2021.projmag.model.ProjectFilter
 
 class ProjectsListActivity : AppCompatActivity() {
+
     private lateinit var itemAdapter: ProjectAdapter
     private lateinit var recyclerView: RecyclerView
-
+    private lateinit var appliedProjects: MutableList<ProjectId>
```

Comment:
The creation of this variable could be done directly here and with val, instead of of a var in onCreate.
```
    private val appliedProjects: MutableList<ProjectId> = ArrayList()
```

Reply:
good point! I've changed that